tensorflow/core/kernels/reshape_util.cc

```diff
@@ -31,6 +31,54 @@ limitations under the License.
 
 namespace tensorflow {
 
+using CPUDevice = Eigen::ThreadPoolDevice;
+
+namespace functor {
+
+template <>
+struct ReshapeSparseTensor<CPUDevice> {
+  Status operator()(OpKernelContext *context, const TensorShape &input_shape,
```

Comment:
`context` is not used and should be removed from the parameter list. We can add it back later if you use it in the GPU implementation.

Reply:
Done.